### PR TITLE
[DEV-4197] Fix uploading files with a unicode filename

### PR DESCRIPTION
**Pull request by Vekkby**
DEV-4197

### CODE REVIEW

**Review by RomanSolianik — APPROVED** [review]

File: lib/jets/controller/params.rb [hunk 1]
```diff
@@ -82,12 +82,14 @@ def body_params
...
   private
-
     def parse_multipart(body)
       boundary = ::Rack::Multipart::Parser.parse_boundary(headers["content-type"])
       options = multipart_options(body, boundary)
       env = ::Rack::MockRequest.env_for("/", options)
-      ::Rack::Multipart.parse_multipart(env) # params Hash
+
+      result = ::Rack::Multipart.parse_multipart(env) # params Hash
+      result['file'].transform_values! { |v| v.instance_of?(String) ? v.force_encoding('UTF-8') : v }
```
Comment (RomanSolianik) [inline lib/jets/controller/params.rb]: pay attention, reductions are still not acceptable

Comment (Vekkby) [reply lib/jets/controller/params.rb]: fixed

Comment (RomanSolianik) [reply lib/jets/controller/params.rb]: it is already approved in any case..

Comment (RomanSolianik) [reply lib/jets/controller/params.rb]: so before merging it (and b2b PR) we should check it on test ENV

Comment (Vekkby) [reply lib/jets/controller/params.rb]: checked, it works

**Pull request merged**